shenyu-client/shenyu-client-grpc/src/main/java/org/apache/shenyu/client/grpc/GrpcClientBeanPostProcessor.java

```diff
@@ -74,7 +83,7 @@ public GrpcClientBeanPostProcessor(final ShenyuRegisterCenterConfig config, fina
         String ipAndPort = props.getProperty("ipAndPort");
         String port = props.getProperty("port");
```

Comment:
It looks weird here. `ipAndPort` and `port`. Maybe `host` and `port` would be better.

Reply:
agree. It can be polished.